src/sensai/featuregen.py

```diff
@@ -195,44 +221,62 @@ def _generateFeatureDict(self, namedTuple) -> Dict[str, Any]:
 
 
 class FeatureGeneratorTakeColumns(RuleBasedFeatureGenerator):
-    def __init__(self, columns: Union[str, List[str]], categoricalFeatureNames: Sequence[str] = (),
-            normalisationRules: Sequence[data_transformation.DFTNormalisation.Rule] = ()):
-        super().__init__(categoricalFeatureNames=categoricalFeatureNames, normalisationRules=normalisationRules)
+    def __init__(self, columns: Union[str, List[str]] = None, exceptColumns: List[str] = None, categoricalFeatureNames: Sequence[str] = (),
```

Comment:
Note: We could use Sequence[str] and default () here; it would simplify the definition below.
(It would also establish a difference to columns where None has an entirely different interpretation.)

Reply:
Good point, I adjusted it